config/jobs/kubernetes-sigs/cluster-api-provider-azure/OWNERS

```diff
@@ -13,6 +13,7 @@ reviewers:
 - juan-lee
 - serbrech
 - vincepri
+- devigned
```

Comment:
```suggestion
- devigned
- juan-lee
- serbrech
- vincepri
```

Reply:
Haha! I was thinking about doing this, but I though it was not very nice to drop the other names down an index.